package/python-kiwi-spec-template

```diff
@@ -208,7 +210,7 @@ Requires:       qemu-img
 Requires:       squashfs-tools
 Requires:       gdisk
 %endif
-%if 0%{?fedora} || 0%{?rhel}
+%if 0%{?fedora} || 0%{?rhel} || 0%{?suse_version} >= 1500
```

Comment:
I talked with the release people and they said we shouldn't hard require dnf. The reason is on the media and products there will be zypper only on suse products. suse products build with kiwi would then require dnf to be present too. This won't be accepted on next submission.

So we can set:

```
Recommends: dnf
```

for the suse package only

Sorry I didn't thought about this when talked on rio

Reply:
Fixed.